pkg/test/harness.go

```diff
@@ -393,6 +402,23 @@ func (h *Harness) Stop() {
 		}
 	}
 
+	if h.bgProcesses != nil {
+		for _, p := range h.bgProcesses {
+			h.T.Log("killing process ", p)
+			err := p.Process.Kill()
+			if err != nil {
+				h.T.Log("background process kill error", err)
```

Comment:
Please log the process name in all places (L414 and L417)  (are command and arguments also available for logging?)